src/utils/normalize-mime-parts.ts

```diff
@@ -146,13 +147,14 @@ export function normalizeMimeParts(
 
 			// remaining non-body, non-enclosure parts are attachments:
 			if (!isBody && type.split('/')[0] !== 'multipart') {
+				const updateDisposition = part.contentId && 'inline'
```

Comment:
If contentId is Present then its an inline attachment but backend is sending disposition  as "attachment" in case document so need to update here
Please let me know if not understood

Reply:
I wonder why this was not needed previously with email-composer.

Reply:
this case came to known with briefCase images only

Reply:
Are we passing content-disposition as inline when add image from briefcase and trigger save draft request?

Reply:
does this happen with legacy also?